content/packages/working-with-a-github-packages-registry/working-with-the-gradle-registry.md

```diff
@@ -192,10 +192,10 @@ To use a published package from {% data variables.product.prodname_registry %},
   ```shell
   repositories {
       maven {
-          url = uri("https://{% ifversion fpt or ghec %}maven.pkg.github.com{% else %}<em>REGISTRY-URL</em>{% endif %}/<em>OWNER</em>/<em>REPOSITORY</em>")
+          url = uri("https://{% ifversion fpt or ghec %}maven.pkg.github.com{% else % REGISTRY-URL{% endif %}/OWNER/REPOSITORY")
```

Comment:
```suggestion
          url = uri("https://{% ifversion fpt or ghec %}maven.pkg.github.com{% else %}REGISTRY-URL{% endif %}/OWNER/REPOSITORY")
```

Reply:
Thanks @ethanpalm 👏🏻 
I literally missed that ;(